docs/autodiff/index.html

```diff
@@ -309,20 +314,22 @@ <h2 id="parameters">Parameters:</h2>
     def add_grad_dependency(self, wrt: &#39;Tensor&#39;, weight: ndarray) -&gt; None:
         self._grad_dependencies.append((wrt, weight))
 
-    def compute_grad(self, debug=False) -&gt; None:
+    def _compute_grad(self, _debug=False) -&gt; None:
```

Comment:
I would split this function it i too long

Reply:
The latest commit addresses this issue, the `_compute_grad` function was split into two functions:
 - `_accumulate_grad_scalar`
 - `_accumulate_grad_matrix`

The new functions are called, depending on the type of the current grad dependency in the for loop - a scalar or a matrix (vectors included).

@shristov-elsys what do you think? Is it good to go now?